core/domain/story_domain.py

```diff
@@ -264,6 +381,49 @@ def __init__(
         self.last_updated = last_updated
         self.version = version
 
+    def validate(self):
+        """Validates various properties of the story object.
+
+        Raises:
+            ValidationError: One or more attributes of story are invalid.
+        """
+        if not isinstance(self.title, basestring):
+            raise utils.ValidationError(
+                'Expected title to be a string, received %s' % self.title)
+
+        if not isinstance(self.description, basestring):
+            raise utils.ValidationError(
+                'Expected description to be a string, received %s'
+                % self.description)
+
+        if not isinstance(self.notes, basestring):
```

Comment:
ditto -- this looks like html.

In general can we also annotate in the class attribute docstrings which fields are intended to be treated as html?